.travis.yml

```diff
@@ -8,10 +8,10 @@ env:
     - GO111MODULE=on
     - CC_TEST_REPORTER_ID=abdad8b2e2ec0bfdd7af57d955c44e9470f4d174e744b824047f3037800f5b40
 
-cache:
-  directories:
-    - $HOME/.cache/go-build
-    - $GOPATH/pkg/mod
+#cache:
+#  directories:
+#    - $HOME/.cache/go-build
+#    - $GOPATH/pkg/mod
```

Comment:
What's the reasoning for removing the caching on the builds? Is this just temporary?

Reply:
removed